tests/Microsoft.Identity.Test.Unit/Kerberos/KerberosSupplementalTicketManagerTests.cs

```diff
@@ -124,6 +124,7 @@ public void GetKrbCred()
             Assert.IsNotNull(krbCred);
         }
 
+#if !NETCOREAPP3_1

```

Comment:
NETCOREAPP3_1 should no longer be defined.

Reply:
yeah, removing